types/aws-param-store/aws-param-store-tests.ts

```diff
@@ -65,14 +66,14 @@ promiseParamResult = getParameter(psName, options);
 promiseParamsResult = getParameters(psNames);
 promiseParamsResult = getParameters(psNames, options);
 
-promiseParamsByPathResult = getParametersByPath(psNames);
-promiseParamsByPathResult = getParametersByPath(psNames, options);
+promiseParamsByPathResult = getParametersByPath(path);
```

Comment:
Yeah this shouldn't have been psNames. I was trying to use the types that are defined for the aws SDK, so looking back at those (https://github.com/aws/aws-sdk-js/blob/a7af3ec024028681a8aa676decf0929d962b9f9a/clients/ssm.d.ts#L4580) this should be `PSParameterName` which is aliased here as `psName`

Reply:
I went with string because PSParameterName infers that you are passing a parameter name which _path_ is not, it is any string that could match any number of parameter names.